Sourcery/Parsing/FileParser.swift

```diff
@@ -61,9 +61,9 @@ final class FileParser {
     ///
     /// - Returns: All types we could find.
     public func parse() -> FileParserResult {
-        let inline = InlineParser.parse(initialContents)
+        let inline = TemplateAnnotationsParser.parseAnnotations("inline", contents: initialContents)
```

Comment:
ah maybe not, I wrote this before I saw how you were using template annotation parser